PopPUNK/network.py

```diff
@@ -642,9 +697,8 @@ def constructNetwork(rlist, qlist, assignments, within_label,
         if weights is not None:
             use_weights = True
         G = add_self_loop(G_df, max_in_vertex_labels, weights = use_weights, renumber = False)
-
+        del G_df
```

Comment:
Again, I'm sure it's a good idea to delete this, but I would just make sure it's the best place to do it so you can't get any unexpected deletes or double deletes. If it's being passed, it shouldn't be deleted in the function taking it

Reply:
Removed in 8fc9763.